ginga/gw/Desktop.py

```diff
@@ -539,11 +540,11 @@ def mypack(w):
             res = []
             for dct in cols:
                 if isinstance(dct, dict):
-                    stretch = dct.get('stretch', 0)
+                    stretch = dct.get('stretch', 0)  # noqa
```

Comment:
I need to take a look at this one.  Let's leave it in but marked (as you've done) for now.